conans/client/tools/files.py

```diff
@@ -247,28 +247,30 @@ def _manage_text_not_found(search, file_path, strict, function_name, output):
         return False
 
 
-def replace_in_file(file_path, search, replace, strict=True, output=None):
+def replace_in_file(file_path, search, replace, strict=True, output=None, encoding_in="auto",
+                    encoding_out="utf-8"):
```

Comment:
I initially wanted to do the same, honestly.
however, it would be a breaking behavior. the current (`1.19.2`) behavior is:

- auto-detect encoding of the input file
- save output file to the **utf-8**

if it's okay to break, I can change to use a single parameter. in that case the default behavior will be:

- auto-detect encoding of the input file
- save output file to the same encoding as input file

Reply:
You can default to None and keep doing the same. We only have to document what is doing the default behavior.

Reply:
changed to use single argument